pipeline/jobs/fill_missing_months.py

```diff
@@ -0,0 +1,71 @@
+from pyspark.sql import DataFrame, functions as F
+
+
+def fill_blanks_with_zeros(
```

Comment:
Here we are adding missing month to the data. Please change the method name to reflect that like "add_missing_months".

Reply:
Thank you. In latest commit I renamed to "add_missing_months". Also added docstrings to this one and other functions as well.